packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -155,6 +155,7 @@ export type Colors = {
 export type ColorsWithModes = Record<ColorSchemeModes, Colors>;
 
 export type ThemeTokens = {
+  name: 'paymentTheme' | 'bankingTheme' | (string & Record<never, never>); // Can be used to watch over state changes between theme without watching over entire theme object
```

Comment:
Should we add a comment addressing why we've added `Record<never, never>`? Would be useful for future contributors when they come across this

Reply:
It's an open issue in TypeScript so it'd be great to link to this in a code comment for clarity:
https://github.com/microsoft/TypeScript/issues/29729

Reply:
done. Also created a type utility that can be reused.